x-pack/plugins/file_data_visualizer/public/application/components/stats_table/components/field_data_expanded_row/keyword_content.tsx

```diff
@@ -5,21 +5,55 @@
  * 2.0.
  */
 
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import type { FieldDataRowProps } from '../../types/field_data_row';
 import { TopValues } from '../../../top_values';
+import { EMSTermJoinConfig } from '../../../../../../../maps/public';
+import { useFileDataVisualizerKibana } from '../../../../kibana_context';
 import { DocumentStatsTable } from './document_stats';
 import { ExpandedRowContent } from './expanded_row_content';
+import { ChoroplethMap } from './choropleth_map';
+
+const COMMON_EMS_LAYER_IDS = [
+  'world_countries',
+  'administrative_regions_lvl2',
+  'usa_zip_codes',
+  'usa_states',
+];
 
 export const KeywordContent: FC<FieldDataRowProps> = ({ config }) => {
-  const { stats } = config;
+  const [EMSSuggestion, setEMSSuggestion] = useState<EMSTermJoinConfig | null | undefined>();
```

Comment:
nit, `EMSSuggestion` should probably start with a lowercase `e`

Reply:
It probably should but it hurts me to have an acronym start with a lowercase letter...